include/onnxruntime/core/providers/vitisai/vitisai_provider_factory.h

```diff
@@ -10,9 +10,12 @@ extern "C" {
 /**
  * \param use_arena zero: false. non-zero: true.
  */
-ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_VITISAI, _In_ OrtSessionOptions* options,
-               const char* backend_type, int device_id, const char* export_runtime_module,
-               const char* load_runtime_module);
+ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_VITISAI, _In_ OrtSessionOptions* options, _In_ const char* opt_str);
```

Comment:
could https://github.com/microsoft/onnxruntime/blob/c6074f3a4b09cb44b46ca4422717deb067015bfa/include/onnxruntime/core/session/onnxruntime_c_api.h#L3527-L3531 be used instead, then one wouldn't need to introduce a new api.

Reply:
We will update it soon.

Reply:
> We will update it soon.
77db8d25a1d2b16ee07598fbbf99186691708256 fix the issue.